packages/wonder-blocks-data/src/util/response-cache.js

```diff
@@ -43,25 +31,49 @@ export class ResponseCache {
         return _default;
     }
 
-    _cache: MemoryCache<any, any>;
+    _hydrationAndDefaultCache: MemoryCache<any, any>;
+    _ssrOnlyCache: ?MemoryCache<any, any>;
 
-    constructor(memoryCache: ?MemoryCache<any, any> = null) {
-        this._cache = memoryCache || new MemoryCache();
+    constructor(
+        memoryCache: ?MemoryCache<any, any> = null,
+        ssrOnlyCache: ?MemoryCache<any, any> = null,
+    ) {
+        this._ssrOnlyCache = Server.isServerSide()
+            ? ssrOnlyCache || new MemoryCache()
```

Comment:
Only want to support this if we are in SSR mode. That includes during testing since that way, we will properly fail still if the test doesn't also mock out Server.isServerSide as it should.